cli/config/config.go

```diff
@@ -33,5 +38,29 @@ func ProcessArgs(cfg interface{}, a *Args, cmd *cobra.Command) {
 
 	envHelp, _ := cleanenv.GetDescription(cfg, nil)
 	cmd.SetUsageTemplate(envHelp + "\n" + cmd.UsageTemplate())
+}
+
+func Prepare(cfg interface{}, a *Args) error {
+	if a.ConfigPath != "" {
+		err := cleanenv.ReadConfig(a.ConfigPath, cfg)
+		if err != nil {
+			return fmt.Errorf("could not read config: %w", err)
+		}
+	}
+	if a.ShareConfigPath != "" {
+		err := cleanenv.ReadConfig(a.ShareConfigPath, cfg)
+		if err != nil {
+			return fmt.Errorf("could not read share config: %w", err)
+		}
+	}
+
+	if a.ConfigPath == "" && a.ShareConfigPath == "" {
+		// When no config file is provided fall back to env vars + default annotations defined on cfg struct.
+		err := cleanenv.ReadEnv(cfg)
+		if err != nil {
+			return fmt.Errorf("could not set up config based on environment variables and struct defaults: %w", err)
+		}
+	}
```

Comment:
This makes sure `cfg` gets initialized with at least some defaults **when no config file is provided** (currently it seems `cfg` can only work with configuration file though because not all the required values are (or can be)  specified directly on the config struct), 

so this gets rid of a `panic` like:
```
goroutine 1 [running]:
go.uber.org/zap/zapcore.multiCore.Enabled(...)
	/go/pkg/mod/go.uber.org/zap@v1.27.0/zapcore/tee.go:68
go.uber.org/zap.(*Logger).check(0x40001af000, 0xff, {0x1b4c044, 0xf})
	/go/pkg/mod/go.uber.org/zap@v1.27.0/logger.go:331 +0x60
go.uber.org/zap.(*Logger).Debug(0x1b4779a?, {0x1b4c044?, 0x185dfc0?}, {0x4000041300, 0x4, 0x4})
	/go/pkg/mod/go.uber.org/zap@v1.27.0/logger.go:238 +0x40
github.com/ssvlabs/ssv/logging.SetGlobalLogger.func1()
	/go/src/github.com/ssvlabs/ssv/logging/global.go:37 +0x2b4
github.com/ssvlabs/ssv/logging.SetGlobalLogger({0x0?, 0x40e4ec?}, {0x0, 0x0}, {0x0, 0x0}, 0x400007ca20)
	/go/src/github.com/ssvlabs/ssv/logging/global.go:100 +0x8d0
github.com/ssvlabs/ssv/cli/operator.setupGlobal()
	/go/src/github.com/ssvlabs/ssv/cli/operator/node.go:466 +0x18c
github.com/ssvlabs/ssv/cli/operator.init.func2(0x2e4ee60, {0x1b3c80e?, 0x4?, 0x1b3c7c6?})
	/go/src/github.com/ssvlabs/ssv/cli/operator/node.go:109 +0xa4
github.com/spf13/cobra.(*Command).execute(0x2e4ee60, {0x40002e9180, 0x1, 0x1})
	/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x63c
github.com/spf13/cobra.(*Command).ExecuteC(0x2e4da40)
	/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/ssvlabs/ssv/cli.Execute({0x1b41a3a?, 0x5?}, {0x400059ea00?, 0x2?})
	/go/src/github.com/ssvlabs/ssv/cli/cli.go:27 +0x88
main.main()
	/go/src/github.com/ssvlabs/ssv/cmd/ssvnode/main.go:20 +0xb0
make: *** [Makefile:119: start-node] Error 2
```
in favor of easily readable error message being logged:
```
2024/10/18 12:34:32 could not create logger: could not set up config based on environment variables: field "Addr" is required but the value is not provided
```